src/components/Class/ClassForm.js

```diff
@@ -296,14 +296,13 @@ function ClassForm({
   ]);
 
   const courses =
-    (data?.Pathways?.data?.pathways[0]?.courses?.map((item) => {
+    data?.Pathways?.data?.pathways[0]?.courses?.map((item) => {
```

Comment:
Replace `data?.Pathways?.data?.pathways[0]?.courses?.map` with `data?.Pathways?.pathwayCourse?.data?.courses?.map`
To make consistency across the component